pkg/util/kubeconfig/kubeconfig.go

```diff
@@ -125,3 +127,18 @@ func GetAuthInfoFromKubeConfig(config *clientcmdapi.Config) *clientcmdapi.AuthIn
 	}
 	return nil
 }
+
+// ToYurtClientSet converts a KubeConfig object to a yurtClient
+func ToYurtClientSet(config *clientcmdapi.Config) (yurtclientset.Interface, error) {
+	overrides := clientcmd.ConfigOverrides{Timeout: "10s"}
+	clientConfig, err := clientcmd.NewDefaultClientConfig(*config, &overrides).ClientConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create yurt client configuration from kubeconfig")
+	}
+
+	client, err := yurtclientset.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create yurt client")
+	}
+	return client, nil
+}
```

Comment:
I think this function is a complete duplicate of `ToClientSet`

Reply:
Almost same, but we need a nodepool client to get info, should you have a better way to do it?